README.md

```diff
@@ -9,34 +9,118 @@ Requirements
 - [Terraform](https://www.terraform.io/downloads.html) 0.12+
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
-## Build the Provider
 
-The instructions outlined below to build the provider are specific to Mac OS or Linux OS only.
+## Cloning the Project
 
-Clone repository to: `$GOPATH/src/github.com/provider/`
+The instructions outlined below are specific to Mac OS or Linux OS only.
+
+If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (please check the [requirements](https://github.com/terraform-providers/terraform-provider-vmc#requirements) before proceeding).
+
+First, you will want to clone the repository to : `$GOPATH/src/github.com/terraform-providers/terraform-provider-vmc`
 
 ```sh
-mkdir -p $GOPATH/src/github.com/provider/
-cd $GOPATH/src/github.com/provider/
-git clone https://github.com/terraform-providers/terraform-provider-vmc.git
+mkdir -p $GOPATH/src/github.com/terraform-providers
+cd $GOPATH/src/github.com/terraform-providers
+git clone git@github.com:terraform-providers/terraform-provider-vmc.git
 ```
 
-Enter the provider directory and build the provider
+## Building the Provider
+
+After the clone has been completed, you can enter the provider directory and build the provider. 
 
 ```sh
-cd $GOPATH/src/github.com/provider/terraform-provider-vmc
+cd $GOPATH/src/github.com/terraform-providers/terraform-provider-vmc
 go get
 go build -o terraform-provider-vmc
 ```
 
-Using the Provider
-----------------------
+After the build is complete, if your terraform running folder does not match your GOPATH environment, you need to copy the `terraform-provider-vmc` executable to your running folder and re-run `terraform init` to make terraform aware of your local provider executable.
+
+
+# Using the Provider
+
+To use a released provider in your Terraform environment, run [`terraform init`](https://www.terraform.io/docs/commands/init.html) and Terraform will automatically install the provider. To specify a particular provider version when installing released providers, see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#version-provider-versions).
+
+To instead use a custom-built provider in your Terraform environment (e.g. the provider binary from the build instructions above), follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-plugins) After placing the custom-built provider into your plugins directory,  run `terraform init` to initialize it.
+
+For either installation method, documentation about the provider specific configuration options can be found on the [provider's website](https://www.terraform.io/docs/providers/vmc/index.html).
+
+
+### Controlling the provider version
```

Comment:
this title is two level below the top level

Reply:
Done